stor_swift/stor_swift/__init__.py

```diff
@@ -0,0 +1,13 @@
+from stor_swift.utils import is_swift_path
+from stor_swift.swift import SwiftPath
+
+
+
+drive = SwiftPath.drive
+
+
+def find_cls_for_path(prefix, path):
```

Comment:
```suggestion
def find_class_for_path(prefix, path):
```

Reply:
Did this for every package in one commit. It's renamed to `class_for_path`